webapp/packages/plugin-projects/src/FolderDialog.tsx

```diff
@@ -178,6 +177,7 @@ export const FolderDialog: DialogComponent<FolderDialogPayload, IFolderDialogRes
             >
               {translate('ui_name')}
             </InputField>
+            {selectProject && <ProjectSelect value={state.projectId} filter={filterProject} onChange={projectId => state.setProjectId(projectId)} />}
```

Comment:
As I remember @Wroud specifically put select before the value, we should sync with him